model/src/main/scala/org/broadinstitute/dsde/rawls/model/DataReferenceModel.scala

```diff
@@ -20,77 +20,6 @@ object DataReferenceModelJsonSupport extends JsonSupport {
-      SNAPSHOT -> stringOrNull(snap.getSnapshot)
-    )
-
-    override def read(json: JsValue) = {
-      json.asJsObject.getFields(INSTANCE_NAME, SNAPSHOT) match {
-        case Seq(JsString(instanceName), JsString(snapshot)) =>
-          new DataRepoSnapshot().instanceName(instanceName).snapshot(snapshot)
-        case _ => throw DeserializationException("DataRepoSnapshot expected")
-      }
-    }
-  }
-
-  // Only handling supported fields for now, resourceDescription and credentialId aren't used currently
-  implicit object DataReferenceDescriptionFormat extends RootJsonFormat[DataReferenceDescription] {
-    val REFERENCE_ID = "referenceId"
-    val NAME = "name"
-    val DESCRIPTION = "description"
-    val WORKSPACE_ID = "workspaceId"
-    val REFERENCE_TYPE = "referenceType"
-    val REFERENCE = "reference"
-    val CLONING_INSTRUCTIONS = "cloningInstructions"
-
-    override def write(description: DataReferenceDescription) = JsObject(
-      REFERENCE_ID -> stringOrNull(description.getReferenceId),
-      NAME -> stringOrNull(description.getName),
-      DESCRIPTION -> stringOrNull(description.getDescription),
-      WORKSPACE_ID -> stringOrNull(description.getWorkspaceId),
-      REFERENCE_TYPE -> stringOrNull(description.getReferenceType),
-      REFERENCE -> description.getReference.toJson,
-      CLONING_INSTRUCTIONS -> stringOrNull(description.getCloningInstructions)
-    )
-
-    override def read(json: JsValue): DataReferenceDescription = {
-      json.asJsObject.getFields(REFERENCE_ID, NAME, DESCRIPTION, WORKSPACE_ID, REFERENCE_TYPE, REFERENCE, CLONING_INSTRUCTIONS) match {
-        case Seq(referenceId, JsString(name), JsString(description), workspaceId, JsString(referenceType), reference, JsString(cloningInstructions)) =>
-          new DataReferenceDescription()
-            .referenceId(referenceId.convertTo[UUID])
-            .name(name)
-            .description(description)
-            .workspaceId(workspaceId.convertTo[UUID])
-            .referenceType(ReferenceTypeEnum.fromValue(referenceType))
-            .reference(reference.convertTo[DataRepoSnapshot])
-            .cloningInstructions(CloningInstructionsEnum.fromValue(cloningInstructions))
-        case _ => throw DeserializationException("DataReferenceDescription expected")
-      }
-    }
-  }
-
-  implicit object DataReferenceListFormat extends RootJsonFormat[DataReferenceList] {
-    val RESOURCES = "resources"
-
-    override def write(refList: DataReferenceList) = JsObject(
-      RESOURCES -> refList.getResources.asScala.toList.toJson
-    )
-
-    override def read(json: JsValue): DataReferenceList = {
-      json.asJsObject.getFields(RESOURCES) match {
-        case Seq(JsArray(resources)) =>
-          new DataReferenceList().resources(resources.map(_.convertTo[DataReferenceDescription]).asJava)
-        case _ => throw DeserializationException("DataReferenceList expected")
-      }
-    }
-  }
-
-
```

Comment:
the model classes referenced by these de/serializers are no longer used or no longer exist